pkg/apis/core/v1beta1/defaults.go

```diff
@@ -15,439 +15,9 @@
 package v1beta1
 
 import (
-	"math"
-	"time"
-
-	corev1 "k8s.io/api/core/v1"
-	"k8s.io/apimachinery/pkg/api/resource"
-	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/runtime"
-	"k8s.io/utils/pointer"
-
-	v1beta1constants "github.com/gardener/gardener/pkg/apis/core/v1beta1/constants"
-	"github.com/gardener/gardener/pkg/utils/timewindow"
 )
 
 func addDefaultingFuncs(scheme *runtime.Scheme) error {
 	return RegisterDefaults(scheme)
 }
```

Comment:
Can we move this function to file where it is used and delete `defaults.go`? I don't see any need to keep `defaults.go` anymore.